content/api/api-events.md

```diff
@@ -7,67 +7,310 @@ author: Lawrence Lane
+        "description": null,
+        "scope": null,
+        "duration": null,
+        "anyCondition": false,
+        "conditions": [],
+        "eventConditions": [],
+        "checkCondition": null,
+        "actions": [],
+        "enabled": true,
+        "deleted": false
+      },
+      "source": null,
+      "title": null,
+      "tags": [],
+      "type": null,
+      "isExternal": false,
+      "processedTime": null,
+      "eventTags": {}
+    },
+    {
+      "id": "9fddc03c-bbea-4005-b8e5-02ad5ef0dac8",
+      "timestamp": "2020-02-08T02:10:00Z",
+      "category": {
+        "id": "1",
+        "name": "INFO",
+        "color": "#0000FF"
+      },
+      "data": {
+        "metricIds": [
+          {
+            "fqn": "aws.ec2.cpuutilization",
+            "id": "1b350b41-e6a6-3a41-8b0a-b9292553fce7"
+          }
+        ],
+        "policyName": "Test-DY-PolicyCreationTime",
+        "metrics": [
+          "aws.ec2.cpuutilization"
+        ],
+        "results": "{\"violating\":true,\"conditions\":[{\"id\":\"0\",\"violating\":true,\"expression\":\"The current value of aws.ec2.cpuutilization is 99.57; this is above the static threshold of 99.\"}],\"metrics\":[\"aws.ec2.cpuutilization\"]}"
+      },
+      "elementId": "24660956-1111-392f-adbe-71c85255bd40",
+      "elementFqn": "511119301106:EC2:us-west-2:i-4b0d3a41",
+      "elementType": "EC2",
+      "elementName": "example-usw2a (10.10.10.6)",
+      "elementLocation": "us-west-2a",
+      "elementTags": {
+        "app": "vpn",
+        "n.analysis.status": "",
+        "Tenant": "internal",
+        "n.collectors": "EC2",
+        "n.backfill.status": "false",
+        "Netuitive": "True",
+        "Name": "example-usw2a"
+      },
+      "policy": {
+        "id": "fe8bd66b-586f-401e-9982-d898aebda30f",
+        "name": "Test-DY-PolicyCreationTime",
+        "description": null,
+        "scope": null,
+        "duration": null,
+        "anyCondition": false,
+        "conditions": [],
+        "eventConditions": [],
+        "checkCondition": null,
+        "actions": [],
+        "enabled": true,
+        "deleted": false
+      },
+      "source": null,
+      "title": null,
+      "tags": [],
+      "type": null,
+      "isExternal": false,
+      "processedTime": null,
+      "eventTags": {}
     }
+  ]
+}
+```
+{{% /expand %}}
 
-  }
-]
+---
+
+## DELETE from /events/delete/policy/{policyId}
+
+{{< button href="https://app.metricly.com/swagger-ui.html#!/events/deleteEventsByPolicyUsingDELETE" theme="danger" >}} DELETE {{< /button >}} Use this endpoint to delete events for a specific policy.
+
+{{% expand "View Method Details." %}}
+
+### Parameters
+
+| Parameter | Parameter Type | Data Type | Description |
+|--------------------|----------------|-----------|----------------------------------------------|
+| policyId | path | string | A policy's unique ID. |
+
+
+### Request URL
+
+```
+https://app.metricly.com/events/delete/policy/{policyId}
```

Comment:
Could we use the copyable code span for request URLs like in the other new API docs?